inventory/group_vars/vagrant

```diff
@@ -27,12 +27,15 @@ nodepool_labels:
 nodepool_providers:
   - name: cicloud
     cloud: cicloud
-    max-servers: 10
+    max-servers: 3
     images:
       - name: ubuntu-xenial
         min-ram: 2048
         diskimage: ubuntu-xenial
         private-key: /var/lib/nodepool/.ssh/id_rsa
+    networks:
+      - name: 'nodepool'
+        public: False
```

Comment:
Also, tip: we also need user-home and username set on the image so that nodepool, otherwise it defaults to jenkins